frameless-runtime/src/lib.rs

```diff
@@ -298,6 +299,61 @@ impl Verifier for OuterVerifier {
 /// The main struct in this module. In frame this comes from `construct_runtime!`
 pub struct Runtime;
 
+// Here we hard-code consensus authority IDs for the well-known identities that work with the CLI flags
+// Such as `--alice`, `--bob`, etc. Only Alice is enabled by default which makes things work nicely
+// in a `--dev` node. You may enable more authorities to test more interesting networks, or replace
+// these IDs entirely.
+impl Runtime {
+
+    /// Aura authority IDs
+    fn aura_authorities() -> Vec<AuraId> {
+        use hex_literal::hex;
+        use sp_application_crypto::ByteArray;
+
+        [
+            // Alice
+            hex!("d43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d"),
+            // Bob
```

Comment:
Should we leave these commented out or add them in?

Reply:
I had left them commented with `--dev` nodes in mind. If we have all six authorities enabled, then a `--dev` node (which has Alice's keys) will only author one block in every six slots.

Reply:
ok great!